opencti-platform/opencti-front/src/private/components/data/connectors/ConnectorWorks.tsx

```diff
@@ -163,11 +169,11 @@ const ConnectorWorksComponent: FunctionComponent<ConnectorWorksComponentProps> =
   const [warnings, setWarnings] = useState<ParsedWorkMessage[]>([]);
   const [tabValue, setTabValue] = useState<string>('Critical');
 
-  const handleOpenDrawerErrors = (errorsList: WorkMessages) => {
+  const handleOpenDrawerErrors = async (errorsList: WorkMessages) => {
     setOpenDrawerErrors(true);
     // TODO : remove examples and use errorList
-    // const parsedList = parseWorkErrors([...(errorsList ?? []), ...examples]);
-    const parsedList = parseWorkErrors(errorsList);
+    const parsedList = await parseWorkErrors([...(errorsList ?? []), ...examples]);
```

Comment:
Please remove all examples from the code.
You can keep them aside for your development, but I'd prefer not seeing this in the release branch.

Reply:
i wanted to keep them here if a reviewer wants to try with these exemples because it's not easy to have real connector errors in local platform. it will be removed before merge, don't worry.

Reply:
ok perfect !

Reply:
Removed